File: 프로그래머스/unrated/181847. 0 떼기/0 떼기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string n_str) {
    string answer = "";
    int count = 0;
    for (int i = 0; i < n_str.size(); i++)
    {
        if (n_str[i] == '0')
            count++;
        else
            break;
    }
    
    if (count >= 1)
        answer = n_str.substr(count,n_str.size());
    else
        answer = n_str;
    
    return answer;
}

File: 프로그래머스/0/181847. 0 떼기/0 떼기.java
class Solution {
    public String solution(String n_str) {
        StringBuilder sb = new StringBuilder();
        
        boolean flag = false;
        for (char c : n_str.toCharArray()) {
            if (c != '0') flag = true;
            
            if (flag) sb.append(c);
        }
        
        return sb.toString();
    }
}